lua/frecency/db.lua

```diff
@@ -0,0 +1,186 @@
+local util = require "frecency.util"
+local const = require "frecency.const"
+local algo = require "frecency.algo"
+
+local ok, sql = pcall(require, "sql")
+if not ok then
+  error "ERROR: frecency-telescope.nvim: sql.nvim (https://github.com/tami5/sql.nvim) is not found. Please install"
+  return
+end
+local s = sql.lib
+
+---@class FrecencyDB: SQLDatabaseExt
+---@field files SQLTableExt
+---@field timestamps SQLTableExt
+---@field workspaces SQLTableExt
+---@field config FrecencyConfig
+local db = sql {
+  uri = vim.fn.stdpath "data" .. "/file_frecency.sqlite3",
```

Comment:
Its important that this is configurable see https://github.com/nvim-telescope/telescope-frecency.nvim/issues/22